src/components/TableParams/index.tsx

```diff
@@ -20,76 +21,80 @@ interface ComponentProps {
 const TableParams: SFC<ComponentProps> = ({className, headers, items}) => {
   const renderBody = (): ReactNode => {
     return (
-      <tbody className={clsx('TableParams__tbody', {...bemify(className, '__tbody')})} data-testid="TableParams__tbody">
-        {items.map(({dataType, description, param, sampleValue}, index) => (
-          <tr
-            key={index}
-            className={clsx('TableParams__tr', {
-              'TableParams__tr--even': index % 2 === 0,
-              ...bemify(className, '__tr'),
-              ...bemify(className, '__tr--odd', index % 2 !== 0),
-            })}
-          >
-            <td
-              className={clsx('TableParams__td TableParams__td--param', {
-                ...bemify(className, '__td'),
-                ...bemify(className, '__td--param'),
+      <S.TableParams>
+        <tbody className={clsx({...bemify(className, '__tbody')})} data-testid="TableParams__tbody">
+          {items.map(({dataType, description, param, sampleValue}, index) => (
+            <S.Tr
+              key={index}
+              className={clsx('TableParams__tr', {
```

Comment:
the point of using styled components is such that you will no longer need to pass in classnames